src/Support/Invoker.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace SergiX44\Nutgram\Support;
+
+trait Invoker
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common processes. Try to leave a link to the process described so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏This trait should be integrated inside the handlergroup class where the feature reside, there is already too much random junk in the `Support` dir 🤷